lib/synapse_pay_rest/http_client.rb

```diff
@@ -123,8 +123,10 @@ def get(path)
     # @raise [SynapsePayRest::Error] subclass depends on HTTP response
     # 
     # @return [Hash] API response
-    def delete(path)
-      response = with_error_handling { RestClient.delete(full_url(path), headers) }
+    def delete(path, payload = nil)
+      execute_arguments = { :method => :delete, :url => full_url(path), :headers => headers, :timeout => 300 }
```

Comment:
I don't think it's problematic but why was this timeout option added?

Reply:
I just followed the behavior in the other client methods that call `execute` directly